src/components/Footer.js

```diff
@@ -1,12 +1,15 @@
 import "./Footer.css";
 import { NavLink } from "react-router-dom";
+import { BsPlusSquare } from "react-icons/bs";
 
 export default function Footer() {
   return (
     <footer className="Footer">
       <nav className="Footer__navigation">
         <NavLink to="/">MyCakes</NavLink>
-        <NavLink to="/add">Plus</NavLink>
+        <NavLink to="/add">
+          <BsPlusSquare size="2rem" />
```

Comment:
I'd avoid adding styles both in css and here. You could just style the NavLinks and the icon would inhenrit the font size. Adriana is doing it like this and it works quite well, maybe you could take a look

Reply:
I agree, using a mix of inline style and css is not a good practice. Will check how Adriana did it, however when I tried to style the NavLinks the icons do not inherit the styles. Also tried to use Context as per instruction on GitHub https://github.com/react-icons/react-icons